devtools/cli/src/main/java/io/quarkus/cli/plugin/PluginCatalogOptions.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.cli.plugin;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import picocli.CommandLine;
+
+public class PluginCatalogOptions {
+
+    @CommandLine.Option(names = {
+            "--project" }, defaultValue = "", paramLabel = "PROJECT", order = 4, description = "Use the project catalog.")
+    boolean project;
```

Comment:
should the default action not be that you install in project if inside a project and user/global if outside?